### PR TITLE
Create colony test

### LINKED ISSUE #987: E2E Test Creating a New Colony
As it says on the tin: write a `cypress` spec that tests updates made to the User profile metadata.

#### Testing

It should cover:
- Creating a new colony
- Creating a new token
- Using an existing token
- Claiming a ENS name for the colony

#### Best practices

Please note that, as a best practice, prefer to add a new `data-test` attribute, to hook onto components, rather then relying on elements/class/ids selectors and filtering from there.

Example:
```js
/*
 * Component (not the actual one, simplified)
 */
const AvatarDropdown = () => (
  <Popover ... >
    <button
      ...
      data-test="avatarDropdown"
    />
  </Popover>
);

/*
 * Test (not the actual one, simplified)
 */
it('Open the Claim Username flow', () => {
  ...
  cy.get('button[data-test="avatarDropdown"]').click();
  ...
});
```

See this example spec: https://github.com/JoinColony/colonyDapp/blob/master/cypress/integration/claim-username.js

#### Resources

A couple of resources to help you along [you're journey](https://i.kym-cdn.com/photos/images/original/000/324/950/24c.jpg):

- [Cypress syntax](https://docs.cypress.io/api/api/table-of-contents.html)
- [Cypress best practices](https://docs.cypress.io/guides/references/best-practices.html)

### PR DESCRIPTION
**Pull request by karolinkas**
## Description

- add cypress commands
- add `data-test` attributes
- add fixtures

Resolves #987

### CODE REVIEW

**Comment by rdig** [comment]
What's the state of this, does it still need work, or is it ready to review ?

**Comment by karolinkas** [comment]
@rdig Even if tests occasionally passed locally, it never actually passed when pushing my changes.

**Comment by rdig** [comment]
Putting this on hold until I see what's what

File: cypress/integration/create-colony.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  it('Confirm user input and process transactions', () => {
+    cy.get('button[data-test="userInputConfirm"]', { timeout: 60000 }).click();
+  });
+
+  it('Sign the transactions', () => {
+    cy.get('ul[data-test="gasStationGroupedTransaction"] > li').each(() => {
+      /*
+       * Confirm the transactions
+       */
+      cy.confirmTx();
```
Comment (rdig) [inline cypress/integration/create-colony.js]: This setup is very nice! :100:

File: cypress/support/commands.js [hunk 2]
```diff
@@ -67,19 +67,47 @@ Cypress.Commands.add(
       ),
 );
 
-Cypress.Commands.add(
-  'checkImage',
```
[the remark this answers is not in the document]
Comment (ceolson01) [reply cypress/support/commands.js]: > Why did you remove this custom command?

Hm.... good question. This was already done when I took this PR over, but I imagine it was not intentional. I'll put it back there. Good eye!

**Pull request merged**